src/main/java/bio/terra/service/filedata/azure/AzureSynapsePdao.java

```diff
@@ -350,7 +350,7 @@ FROM OPENROWSET(
                   FROM OPENROWSET(BULK '<parquetFileLocation>',
                                 DATA_SOURCE = '<datasource>',
                                 FORMAT='PARQUET') WITH (
-                                  <columns:{c|[<c.name>] <if(c.requiresTypeCast)>varchar(8000)<else><c.synapseDataType><endif>
+                                  <columns:{c|[<c.name>] <if(c.requiresTypeCast)>varchar(max)<else><c.synapseDataType><endif>
```

Comment:
I'm having some trouble understanding this file - Is this code not used for CSV ingests? I ask because I see the replacements of 8000 with max, and I know you said that that won't work for CSV ingests.

Reply:
Ah yeah, sorry, this has to do with _where_ synapse doesn't support `varchar(max)`.  It actually works great in most cases _except_ when trying to use the `'CSV'` format with `OPENROWSET`.  It's really just this one branch of code in Synapse (the CSV parser) that doesn't work with varchar max

Reply:
Thank you!